Edits to training guide

rachelsun97 opened the pull request:
Removed deprecated functions and added example of custom units dataframe to edit_attributelists.Rmd

dmullen17 commented on workflows/edit_eml/edit_attributelists.Rmd:
````diff
@@ -105,13 +113,22 @@ If you have units that are not in the standard EML unit list, you will need to b
+  description = c('siemens per meter', 'decibar'),
+  
+  stringsAsFactors = FALSE)
 ```
````
This is looking really good.  I think you could add a line where you insert this into the EML doc.

dmullen17 replied:
I actually see that in section 4.3.4 but I think it makes sense to just move it up to the bottom of this section so it's more self-contained

dmullen17 commented:
@rachelsun97 I really like the documentation you wrote for `recover_failed_submission`.  Could you move this from editing eml to [here](https://nceas.github.io/datateam-training/reference/adc-web-submissions.html) instead? I think that's a more appropriate spot for it.  Otherwise looks great!

rachelsun97 commented:
@dmullen17 I moved ```recover_failed_submission``` to the ```ADC web submissions``` section, and adjusted the custom units section according to your earlier comments

dmullen17 commented:
@rachelsun97 this all looks good to me!  I'll merge it in now.
